README.md

```diff
@@ -22,6 +24,31 @@ You can submit a talk or a workshop proposal [here](https://github.com/parisp2p/
 
 _Fill an issue [here](https://github.com/parisp2p/events/issues)._
 
+## Chapters
+
+<table>
+  <tr>
+    <td align="center">
+      <a href="https://p2p.paris/">
+        <img src="https://p2p.paris/img/paris_p2p.svg" alt="Paris P2P" height="200px" /><br />
+        Paris P2P
+      </a>
+    </td>
+    <td align="center">
+      <a href="https://github.com/parisp2p/community/issues/10">
+        <img src="https://via.placeholder.com/200x200?text=Coming%20soon" alt="Lyon P2P" height="200px" /><br />
```

Comment:
we do have the logo now, isn't it ?

Reply:
yep: https://github.com/francep2p/community/tree/master/assets/logos/lyon/v2

fixed!